### ISSUE TITLE
QField crashes after inserting more than 10 objects into a project

### ISSUE THREAD

**Issue by gemmos** [opengisch/QField#876]
I have imported a QGIS project to an Android device (T-SM365 Samsung Galaxy Tab Active, Android Version: 5.1.1) via the QFieldSync Plugin. The project contains several layers (11 layers for mapping different groups of species in the same location and 3 layers to indicate the areas where the data will be collected). After importing the project it seemed to work fine until I created about 10 new objects in one of the layers. After creating the 11th object, the App crashed and could not open the project again. It starts to load the project, but after a few moments the App crashes again. All the created objects are still saved in the Project and I was able to retrieve the data in QGIS. Obviously, the project was designed to contain a whole lot more objects... up to 100-300 data points per layer. 

After restarting the App, and the tablet several times it still could not load the project without crashing shortly after. I also tried deleting an re-importing the empty project to the tablet, but the same crash happens again after creating more than 10 objects.

The QField Version is 1.4.4

## Additional information:
I tried to test the same project on my Smartphone (Xaomi Redmi 4, Android 6.0.1) with the same Version of QField and it worked perfectly so far.
But for my job it would be essential to work on the Galaxy Tab Active.

I am pretty new to QGIS and if any further information would be useful, I will be happy to provide it, if you can give me a short description on how to get it.

**Comment by Sgachet** [comment]
Hi,

I have more or less the same kind of problem with the same model of tablet. 
When I want to activate 2 layers, the application stops. 
Same project on another tablet and on a phone, everything works as expected.

**Model** : T-SM365 Samsung Galaxy Tab Active, 
**Android** Version: 5.1.1
**Qfield Version** : 1.3.4
**Plugin Qfield Version** : 3.1.6

Thanks for the help

**Comment by gemmos** [comment]
Just a short comment about the issue to whomever it might concern:

The workaround I used for this project was to separate the different layers for the different species-groups into several smaller projects on the tablet. Certainly not the most comfortable method, but it worked.

After an upgrade to the Qfield Version 1.7.8, I was curious to see if the issue still exists.
I imported the old project with all the layers (including the more than 2500 recorded data points) and the project loads pretty slowly (takes about 2,5 minutes before anything is displayed) and when several layers are displayed it lags significantly when creating new points in one of the displayed layers, but other than that the app works fine.
